pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/json/ImmutableJsonIndexReader.java

```diff
@@ -62,6 +66,7 @@ public class ImmutableJsonIndexReader implements JsonIndexReader {
   private final int _version;
   private final StringDictionary _dictionary;
   private final BitmapInvertedIndexReader _invertedIndex;
+  private final long _numFlattenedDocs;
```

Comment:
(minor) Is this used?

Reply:
This is used in the other PR (https://github.com/apache/pinot/pull/12683) for exclusive predicate handling. May have accidentally forgotten when splitting. If not an issue, would let it be part of this pr itself since #12683 should be merged soon too..